client/jetpack-connect/plans-landing.jsx

```diff
@@ -76,6 +79,10 @@ class PlansLanding extends Component {
 		this.storeSelectedPlan( null );
 	};
 
+	handleHelpButtonClick = () => {
+		this.props.recordTracksEvent( 'calypso_jpc_help_link_click' );
```

Comment:
Is it worth adding some parameter to know where the tracks event comes from? Plans, activate, etc?

Reply:
Sounds like a good idea to me. Currently we can narrow down by using the `pageView` events when building funnels, but it would be useful to be able to directly filter. I've opened an issue for this one: #18658. Thanks!